README.md

```diff
@@ -153,6 +153,52 @@ they fail due to indirect failure from the request
 at the head of the pipeline. This does not apply to
 idempotent requests with a stream request body.
 
+<a name='stream'></a>
+#### `client.stream(opts, cb(err, data))`
+
+A faster version of `'request'`. Not promisified.
+
+Unlike `request` this method expects `callback`
+to return a `Writable` which the response will be
+written to. This improves performance by avoiding
+creating an intermediate `Readable` when the user
+expects to directly pipe the response to a 
+`Writable`.
+
+```js
+const { Client } = require('undici')
+const client = new Client(`http://localhost:3000`)
+const fs = require('fs')
+
+const dst = fs
+  .createWriteStream('destination.raw')
+  .on('finish', function () {
+    console.log('file success')
+  })  
+  .on('error', function (err) {
+    console.log('file error', err)
+  })
+
+client.stream({
+  path: '/',
+  method: 'GET'
+}, function (err, data) {
+  if (err) {
+    dst.destroy(err)
+    return
+  }
+  const {
+    statusCode,
+    headers
+  } = data
+
+  console.log('response received', statusCode)
+  console.log('headers', headers)
+
+  return dst
```

Comment:
Most developers would not know how to check when the file has been fully stored on disk. Can you expand this example using `finished`?

Reply:
Updated. PTAL.